exe-common/src/config.rs

```diff
@@ -192,6 +192,7 @@ pub mod net {
     pub struct Config {
         pub genesis: HeaderHash,
         pub genesis_prev: HeaderHash,
+        pub k: u32,
```

Comment:
I think we agreed on something a bit longer (my suggestion was something like 'epoch_stability_depth' IIRC)